cmd/iam_apikey_create.go

```diff
@@ -0,0 +1,48 @@
+package cmd
+
+import (
+	"strings"
+
+	"github.com/exoscale/egoscale"
+	"github.com/spf13/cobra"
+)
+
+type apiKeyCreateItemOutput egoscale.APIKey
```

Comment:
For custom output you should not rely on the raw API struct but rather create a dedicated `apiKeyShowOutput` structure as it's done in every other show/list command implementation: this way, we get to control which resources properties and how they are exposed to the users regardless of how they are named in the API.

You can get inspiration from the `affinitygroup show` command implementation: https://github.com/exoscale/cli/blob/master/cmd/affinitygroup_show.go#L11-L21